Source/WarpX.H

```diff
@@ -1322,8 +1370,12 @@ private:
     amrex::Vector<            std::unique_ptr<amrex::MultiFab>      > F_fp;
     amrex::Vector<            std::unique_ptr<amrex::MultiFab>      > G_fp;
     amrex::Vector<            std::unique_ptr<amrex::MultiFab>      > rho_fp;
+    amrex::Vector<            std::unique_ptr<amrex::MultiFab>      > rho_fp_temp;
```

Comment:
Could you make these variables members of the `HybridPICModel` instead? 
This would provide more encapsulation and make it clearer which data members go along with which solver/scheme.

Reply:
Also: these new data structures may cause problems when doing load-balancing, unless they are redistributed along with the other arrays.
It may be good to raise an error when hybrid is used along with load-balancing.